testing/upload_validation.ts

```diff
@@ -1269,6 +1278,16 @@ function validateFormulas(schema: z.ZodObject<any>) {
         path: ['defaultAuthentication.networkDomain'],
       },
     )
+    .superRefine((data, context) => {
+      const buildingBlockCount = (data.syncTables?.length as number || 0) + (data.formulas?.length as number ?? 0) +
```

Comment:
I think I'd just simplify this and say you can have at most 100 formulas, 100 sync tables, and 100 column formats. Little easier to understand, document, and implement.

Reply:
Added. I was wondering what places I had to document this in? I added something to the changelog. Should I add something to the SDK docs as well (maybe the troubleshooting section?)

Reply:
When @ekoleda-codaio is back he can advise on the best place in the documentation to mention these.

Reply:
As discussed in chat, my advice is to actually not document the limits for now. As long as the error message is clear, it will be just as informative and guaranteed to be up-to-date.